### PR TITLE
move-audience-out-as-its-part-of-payload

### PR DESCRIPTION
**Pull request by nehaabrol87**
@jonathanbarton  @morrissinger 
Want to try without the audience.

### CODE REVIEW

File: src/server/middleware/authentication.js [hunk 1]
```diff
@@ -20,10 +19,8 @@ function parseHeader(header) {
 }
 
 function jwtVerify(req, res, header, done) {
-  console.log(`Secret ${jwtSecret}`);
-  console.log(`Audience ${jwtAudience}`);
   const token = parseHeader(header);
-  jwt.verify(token, jwtSecret, { aud: jwtAudience }, (err, decoded) => {
+  jwt.verify(token, jwtSecret, (err, decoded) => {
```
Comment (nehaabrol87) [inline src/server/middleware/authentication.js]: Yes!! Exactly.
Either that or it needs to be included in payload as aud.
So after merging this we I will create a token with aud key in the payload .

**Comment by morrissinger** [comment]
Can we merge this?

**Comment by nehaabrol87** [comment]
Haaa I was waiting for one of your thumbsup.Let's merge.

File: src/server/middleware/authentication.js (hunk 1 above)
Comment (jonathanbarton) [inline src/server/middleware/authentication.js]: 👍

**Comment by morrissinger** [comment]
👍 🚢 🚢 🚢

**Pull request merged**